.github/workflows/build.yml

```diff
@@ -152,3 +152,28 @@ jobs:
         allow_issue_writing: false
         fail_action: true
         cmd_options: '-z "-configFile /zap/wrk/options.prop"'
+
+  deploy_test:
+    name: Deploy to test environment
+    needs: [build, validate_terraform]
```

Comment:
Do we need to include deploy_dev as dependency here?.

Reply:
<img width="733" alt="image" src="https://user-images.githubusercontent.com/2041280/144817848-110232ad-d3a2-44d9-9548-c55ae242be79.png">